src/ZMQ.jl

```diff
@@ -437,10 +437,13 @@ end
 
 # Convert message to string (copies data)
 unsafe_string(zmsg::Message) = Compat.unsafe_string(pointer(zmsg), length(zmsg))
-if VERSION < v"0.5-dev+4341"
-    bytestring(zmsg::Message) = unsafe_string(zmsg)
-else
-    @deprecate bytestring(zmsg::Message) unsafe_string(zmsg::Message)
+if isdefined(Base, :bytestring)
+    import Base: bytestring
+    if VERSION < v"0.5-dev+4341"
+        bytestring(zmsg::Message) = unsafe_string(zmsg)
```

Comment:
On 0.5, seems like the analogous thing would be to define`String(zmsg::Message)`?